.github/workflows/branch-protection.yml

```
@@ -0,0 +1,131 @@
+
+on:
+  workflow_dispatch:  # Ручной запуск
+  push:
+    branches: [main]
+    paths: ['.github/workflows/branch-protection.yml']
+
+jobs:
+  setup-branch-protection:
+    runs-on: ubuntu-latest
+    if: github.repository == 'Abilka94/WoW-Reg-Legion-Bot'
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)